music21/meter/core.py

```diff
@@ -418,6 +418,13 @@ def __init__(self, value=None, partitionRequest=None):
 
     # SPECIAL METHODS #
 
+    def __eq__(self, other) -> bool:
+
+        if not super().__eq__(other):
+            return False
+
+        return True
+
```

Comment:
This is also not needed.   If you're just returning the value of a `super()` call, then there's no point in subclassing.

Reply:
Ok thanks. True in so far as the actual test is not needed ... but the method is not expendable. Something is needed here, perhaps just to give MeterTerminal some notion of equality. FWIW empty equalityAttributes tuple (`equalityAttributes = ()`) doesn't work.